presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -346,7 +346,7 @@ public CallExpression comparison(OperatorType operatorType, RowExpression left,
     {
         private final TypeProvider types;
         private PlanNode source;
-        private Map<VariableReferenceExpression, Aggregation> assignments = new HashMap<>();
+        private ImmutableMap.Builder<VariableReferenceExpression, Aggregation> assignments = ImmutableMap.builder();
```

Comment:
I would use a LinkedHashMap here.  And also add a comment explaining why we are about the order.

Reply:
Done